api-spec/examples.md

```diff
@@ -103,7 +123,7 @@ Response:
 ```
 
 This tells the client to post to the search endpoint with a body of:
-```
+```json
```

Comment:
This is not valid JSON due to the POST ... header

Reply:
Yeah, but I'm pretty sure it's just a style hint so it shouldn't hurt anything and makes it look prettier right?

Reply:
Seems like that. Sometimes it makes things worse, but here it doesnt.